src/commands/addBinding/settingSteps/eventHub/EventHubNameStep.ts

```diff
@@ -11,6 +11,6 @@ export class EventHubNameStep extends StringPromptStep {
     public shouldPrompt(context: IEventHubWizardContext & IBindingWizardContext): boolean {
         // If the user decides to create a new app setting, `EventHubListStep` will take care of prompting
         // Otherwise, prompt to manually enter the name of the event hub using this step
-        return !context.namespaceName && !context.eventhubname;
+        return !context.eventHubsNamespace && !context.eventhubname;
```

Comment:
Nit: Camel case?

`eventhubname` => `eventHubName`

Reply:
There's a note in IEventHubWizardContext

```
  /**
     * NOTE: The name of this variable should not change. It matches the name of the binding setting written to function.json
     */
    eventhubname?: string;
```